content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -24,16 +24,13 @@ Nested reads allow you to read related data from multiple tables in your databas
 - Use [`include`](/reference/api-reference/prisma-client-reference#include) <span class="api"></span> to include related records, such as a user's posts or profile, in the query response.
 - Use a nested [`select`](/reference/api-reference/prisma-client-reference#select) <span class="api"></span> to include specific fields from a related record. You can also nest `select` inside an `include`.
 
-The following example returns a single user _and_ that user's posts:
+The following example returns a single user and that user's posts:
 
 <CodeWithResult>
 <cmd>
 
-```ts highlight=5-7;normal
-const getUser = await prisma.user.findUnique({
-  where: {
-    id: 19,
-  },
+```ts
+const getUser = await prisma.user.findFirst({
```

Comment:
why the switch from `findUnique` to `findFirst`?

Reply:
For the nested reads, we intentionally switched it to `findFirst` to simplify the queries, which doesn't require any filters. This makes it easy to highlight the important parts of the query to perform nested read. It also removes the need for manually adding highlights for the parts of the code that change.